source/c_modelling_with_live_site.rst

```diff
@@ -138,6 +139,50 @@ When choosing a **Constant** type of precipitation, the rain intensity (in mm/h)
 
 The option **Radar** is only available in the Netherlands and uses historical rainfall data that is based on radar rain images. Providing temporally and spatially varying rain information. The Dutch Nationale Regenradar is available for all Dutch organisations that have the NRR module. On request, the information from other radars can be made available to 3Di as well. In order to apply this type of rain a historical time frame needs to be set. 
 
+The **Forecast** option gives you the option to add forecast data from several sources to the model.
+Currently, we support forecast data from ICON Global, ICON EU, ICON-D2 and QPE Forecast.
+
+Below is a table showing information about the ICON forecast. It shows the difference in time between the forecast moment in UTC and a few derivates thereof, including forecast finished UTC, run moment ansible (CEST/UTC+2) and forecast horizon (hours).
```

Comment:
@GeOdin deze tabel begrijp ik niet helemaal. Ik begrijp vooral niet wat het doel ervan is (wat wil je ermee vertellen). Dat komt denk ik o.a. doordat ik niet weet wat 'run moment ansible' en 'forecast horizon' betekenen. Kan je dat anders formuleren zodat ik het wel begrijp?